spring-integration-syslog/src/main/java/org/springframework/integration/syslog/inbound/TcpSyslogReceivingChannelAdapter.java

```diff
@@ -61,7 +71,7 @@ public String getComponentType() {
 	protected void onInit() {
 		super.onInit();
 		if (this.connectionFactory == null) {
-			this.connectionFactory = new TcpNioServerConnectionFactory(this.getPort());
+			this.connectionFactory = new TcpNioServerConnectionFactory(super.getPort());
```

Comment:
I think we don't need this change. 
We have similar structure in the `UdpSyslogReceivingChannelAdapter` below but you don't override the `onInit()` logic there.